docs/src/pages/components/actionmenu.mdx

```diff
@@ -0,0 +1,265 @@
+---
+name: Action Menu
+route: /components/actionmenu
+---
+
+import ActionMenu from '@pluralsight/ps-design-system-actionmenu'
+import { Intro, Types, Usage } from '../../components/component'
+
+# Action Menu
+
+<Intro>
+  Used to perform an action, or to navigate from a list of options
+</Intro>
+
+## Examples
+
+### Icons
+
+Use <code>ActionMenu.Icon</code> to add context and recognition to action menu items.
+
+```typescript
+import React from 'react'
+import ActionMenu from '@pluralsight/ps-design-system-actionmenu'
+import { PencilIcon, ChannelIcon, PlayIcon } from '@pluralsight/ps-design-system-icon'
+
+function Example () {
+  return <ActionMenu >
+      <ActionMenu.Item>
+        <ActionMenu.Icon marginLeft>
+          <PencilIcon />
+        </ActionMenu.Icon>
+        One item
+      </ActionMenu.Item>
+      <ActionMenu.Item>
+        <ActionMenu.Icon marginLeft>
+          <ChannelIcon />
+        </ActionMenu.Icon>
+        Two item
+      </ActionMenu.Item>
+      <ActionMenu.Item>
+        <ActionMenu.Icon marginLeft>
+          <PlayIcon />
+        </ActionMenu.Icon>
+        Three item
+      </ActionMenu.Item>
+    </ActionMenu>
+}
+export default Example
+```
+
+### Ellipsis
+Use <code>ActionMenu.Ellipsis</code> to add ellipsis to action menu items.
+
+```typescript
+import React from 'react'
+import ActionMenu from '@pluralsight/ps-design-system-actionmenu'
+function Example(){
+  return <ActionMenu  >
+      <ActionMenu.Item>
+        <ActionMenu.Ellipsis>
+          One item that has text that goes on forever and onward into the
+          universes yet to be
+        </ActionMenu.Ellipsis>
+      </ActionMenu.Item>
+      <ActionMenu.Item>
+        <ActionMenu.Ellipsis>
+          Another item that takes a long time to explain in the context of
+          everything that is in a line.
+        </ActionMenu.Ellipsis>
+      </ActionMenu.Item>
+    </ActionMenu>
+}
+export default Example
+```
+
+### Dividers
+Dividers can be useful to separate similar actions. Dividers are applied 
+at the list level, below the assigned list item.
+
+```typescript
+import React from 'react'
+import ActionMenu from '@pluralsight/ps-design-system-actionmenu'
+function Example() {
+  return <ActionMenu >
+      <ActionMenu.Item>
+        One item
+      </ActionMenu.Item>
+      <ActionMenu.Divider />
+      <ActionMenu.Item>
+        Two item
+      </ActionMenu.Item>
+      <ActionMenu.Item>
+        Three item
+      </ActionMenu.Item>
+      <ActionMenu.Divider />
+      <ActionMenu.Item>
+        Four item
+      </ActionMenu.Item>
+    </ActionMenu>
+}
+export default Example
+```
+
+### Nesting
```

Comment:
Style-wise, I'd put a line break after this to be consistent with other mdx

Reply:
Cool